bitcoin/contrib/test.sh

```diff
@@ -12,11 +12,16 @@ fi
 cargo --version
 rustc --version
 
-# Work out if we are using a nightly toolchain.
+# Some tests require certain toolchain types.
 NIGHTLY=false
+STABLE=true
 if cargo --version | grep nightly; then
+    STABLE=false
     NIGHTLY=true
 fi
+if cargo --version | gre beta; then
```

Comment:
Actually no, because this is in an `if` command it is probably just always failing, so the `if` is never taken.

But I don't know bash well enough to be certain.

Reply:
@apoelstra is correct.